inference-engine/src/inference_engine/src/cpp/ie_executable_network.cpp

```diff
@@ -106,3 +108,58 @@ ExecutableNetwork::operator bool() const noexcept {
     return !!_impl;
 }
 }  // namespace InferenceEngine
+
+namespace ov {
+namespace runtime {
+ExecutableNetwork::ExecutableNetwork(const std::shared_ptr<SharedObject>& so,
+                                     const std::shared_ptr<ie::IExecutableNetworkInternal>& impl)
+    : _so{so},
+      _impl{impl} {
+    IE_ASSERT(_impl != nullptr);
+}
+
+std::shared_ptr<const Function> ExecutableNetwork::get_function() const {
+    EXEC_NET_CALL_STATEMENT(return std::const_pointer_cast<const Function>(_impl->GetExecGraphInfo()));
+}
+
+std::vector<std::shared_ptr<ngraph::op::v0::Parameter>> ExecutableNetwork::get_parameters() const {
+    EXEC_NET_CALL_STATEMENT(return _impl->GetExecGraphInfo()->get_parameters());
```

Comment:
can we provide an alternative implementation if `GetExecGraphInfo` is not implemented like for GNA? E.g. manually create params / results

Reply:
Added default GetExecGraphInfo implementation #7352